src/app.ts

```diff
@@ -0,0 +1,69 @@
+import { MikroORM, RequestContext } from '@mikro-orm/core';
+import { PostgreSqlDriver } from '@mikro-orm/postgresql';
+import express, { Application } from 'express';
+import ormConfig from './orm.config';
+import { TrialService } from './components/trial.component/trial.component';
+import { CureWikiService } from './components/curewiki.component/curewiki.component';
+import { NextFunction, Request, Response } from 'express';
+import Container from './utils/helpers/container';
+import { AIService } from './components/ai.component/ai.component';
+import { useExpressServer } from 'routing-controllers';
+import { TrialController } from './controllers/database/trial.controller';
+import 'reflect-metadata';
+
+export class App {
+  host: Application;
+  public orm: MikroORM<PostgreSqlDriver>;
+  trialService: TrialService;
+  cureWikiService: CureWikiService;
+  aiService: AIService;
+  public app: express.Application;
+
+  constructor() {
+    this.host = express();
+    this.host.use(express.json());
+
+    this.host.use((req: Request, res: Response, next: NextFunction) => {
+      console.log(req.method, req.url);
+      next();
+    });
+
+    this.initializeControllers([TrialController]);
+
+    this.trialService = new TrialService();
+    this.cureWikiService = new CureWikiService();
+    this.aiService = new AIService();
```

Comment:
extract into separate function or even container. This might change over time and some can also be mocked in tests.